discovery-provider/integration_tests/tasks/entity_manager/test_user_entity_manager.py

```diff
@@ -227,6 +249,20 @@ def test_index_valid_user(app, mocker):
                 )
             },
         ],
+        "CreateUser3Tx": [
+            {
+                "args": AttributeDict(
+                    {
+                        "_entityId": USER_ID_OFFSET + 3,
+                        "_entityType": "User",
+                        "_userId": USER_ID_OFFSET + 3,
+                        "_action": "Create",
+                        "_metadata": f'{{"data": {create_user3_json}}}',
```

Comment:
no cid prop?

Reply:
do we need CID still? for v2 @michellebrier

Reply:
oh yea for that CID table right?